samples/net/zperf/src/zperf_tcp_receiver.c

```diff
@@ -143,21 +145,36 @@ void zperf_tcp_receiver_init(const struct shell *shell, int port)
 			return;
 		}
 
-		ret = zperf_get_ipv4_addr(shell, MY_IP4ADDR,
-					  &in4_addr_my->sin_addr);
-		if (ret < 0) {
-			shell_fprintf(shell, SHELL_WARNING,
-				      "Unable to set IPv4\n");
-			return;
+		if (MY_IP4ADDR && strlen((const char *)MY_IP4ADDR)) {
```

Comment:
No need to call strlen() here, just do `MY_IP4_ADDR[0]`

Reply:
Thanks Jukkar,
With MY_IP4ADDR[0], compiled error is happening.
zephyr/samples/net/zperf/src/zperf_udp_receiver.c:329:30: warning: pointer of type ‘void *’ used in arithmetic [-Wpointer-arith]
   if (MY_IP4ADDR && MY_IP4ADDR[0]) {
                              ^
zephyr/samples/net/zperf/src/zperf_udp_receiver.c:329:30: error: void value not ignored as it ought to be
   if (MY_IP4ADDR && MY_IP4ADDR[0]) {

Reply:
If MY_IP4ADDR is NULL,  it's not allowed to access it with MY_IP4ADDR[0].